### PR TITLE
Removed unused repository paths from repositories.config

### PR DESCRIPTION
**Pull request by zharro**
Fixes [#1827](https://github.com/nunit/nunit/issues/1827): removed all but NUnitFramework paths.

### CODE REVIEW

File: packages/repositories.config [hunk 1]
```diff
@@ -1,18 +1,7 @@
...
   <repository path="../src/NUnitFramework/testdata/packages.config" />
   <repository path="../src/NUnitFramework/tests/packages.config" />
-  <repository path="..\src\NUnitEngine\Addins\nunit.v2.driver.tests\packages.config" />
-  <repository path="..\src\NUnitEngine\mock-v2-assembly\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.core.engine\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.engine.tests\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.engine\packages.config" />
-  <repository path="..\src\NUnitEngine\nunit.v2.driver.tests\packages.config" />
   <repository path="..\src\NUnitFramework\testdata\packages.config" />
   <repository path="..\src\NUnitFramework\tests\packages.config" />
```
Comment (jplebre) [inline packages/repositories.config]: Odd, is this supposed to be repeated, but with inverted `/` ? seems a bit odd to me never noticed this before.

**Comment by jplebre** [comment]
Quick question should this also include all the packages from the other projects in the folder?
more notably `NUnitFramework/framework`

**Comment by rprouse** [comment]
Yes, I think it should include the `packages.config` in the framework. I am not sure why it wasn't automatically added.

**Comment by zharro** [comment]
Looks like Linux build failed after removing Linux slashes. I'll try to figure out the reason.

**Comment by ChrisMaddock** [comment]
@zharro - The test error looks unrelated to your changes.

They're occasionally a little flakey. I've restarted the build, hopefully they'll pass the second time, and we can merge!

**Comment by rprouse** [comment]
All passing now. Looks good, merging. Thanks.

**Pull request merged**